developer_docs/testing.md

```diff
@@ -0,0 +1,472 @@
+# Testing
+For an initial basic overview of testing for React apps, [you can read what the React developers have to say about it](https://reactjs.org/docs/testing.html).
+
+We are testing React components by rendering the component trees in a simplified test environment and making assertions on what gets rendered and what functions get called.
+
+Many files still don't have tests, so if you're looking to get started as a contributor, this would be a great place to start!
+
+## What's in this document
+- [Testing dependencies](#testing-dependencies)
+- [Useful testing commands](#Useful-testing-commands)
+- [Why write tests](#Why-write-tests)
+- [When to run tests](#When-to-run-tests)
+- [Writing a test](#Writing-a-test)
+- [Files to be aware of](#Files-to-be-aware-of)
+- [Testing plain components](#Testing-plain-components)
+- [Testing Redux](#Testing-Redux)
+- [How to handle API calls in tests](#How-to-handle-API-calls-in-tests)
+- [Some more background on tests](#Some-more-background-on-tests)
+- [Internationalization](#internationalization)
+- [Tips](#Tips)
+- [More resources](#More-resources)
+- [References](#References)
+
+
+## Testing dependencies
+1. [Jest](https://jestjs.io/)
+2. [react-testing-library](https://testing-library.com/docs/react-testing-library/intro/)
+3. [redux-mock-store](https://github.com/reduxjs/redux-mock-store)
+
+## Useful testing commands
+Run the whole test suite
+```
+npm run test
```

Comment:
Usually for shell commands in documentation I preface them with a `$`

Reply:
fixed :-) thank you!